src/main/scala/firrtl/transforms/RemoveKeywordCollisions.scala

```diff
@@ -6,6 +6,7 @@ import firrtl._
 
 import firrtl.analyses.InstanceGraph
 import firrtl.annotations.{Named, CircuitName, ModuleName, ComponentName}
+import firrtl.annotations.{Target, GenericTarget}
```

Comment:
I don't see that this import is necessary. Nothing else in file has changed in order to reference it. Was something missed here?